src/documents/serialisers.py

```diff
@@ -186,6 +186,18 @@ def create(self, validated_data):
     def update(self, instance, validated_data):
         if "set_permissions" in validated_data:
             self._set_permissions(validated_data["set_permissions"], instance)
+        if "owner" in validated_data and "name" in self.Meta.fields:
```

Comment:
This is more about `OwnedObjectSerializer` than this particular change, but it would allow a user to create something for another user, by setting `"owner"` to another valid user pk, right?

Reply:
yes e.g.
<img width="595" alt="Screenshot 2023-03-08 at 2 42 36 PM" src="https://user-images.githubusercontent.com/4887959/223868103-93df627e-3b57-42c9-9f63-4ba11add8d25.png">